twitchpopups.htm

```diff
@@ -149,9 +157,33 @@ <h1 id="popuptext"></h1>
         $("#popuptext").animate({"opacity":0, "margin-left":"50px"}, 700);
     }
     
+    function formatEmotes(message, emotes) {
+		  //parse the message for html and remove any tags
+      var newMessage = $($.parseHTML(message)).text().toUpperCase().split("");
+
+			//replace any twitch emotes in the message with img tags for those emotes
+      if(twitchEmotes){
+        for (var emoteIndex in emotes) {

```

Comment:
If emotes is an array

```
emotes.forEach((e, i) => {

});
```

Reply:
It's a hash set or dictionary or something who knows what its called in js but anyway this didn't work so I just left it as it is